scripts/funcTests/runFuncTests.sh

```diff
@@ -89,6 +86,30 @@ done
 
 # Display .NET CLI info
 $DOTNET --info
+if (( $? )); then
+	echo "DOTNET --info failed!!"
+	exit 1
+fi
+
+# Get recommended version for bootstrapping testing version
+
+echo "cli/dotnet-install.sh --install-dir cli --runtime dotnet --channel 5.0 -nopath"
```

Comment:
I imagine this matches the tfms our tests are targetting?

Reply:
That right! We have test projects targeting netcoreapp3.1 and/or net5.0, and dotnet.integration.tests targeting net6.0.
So we install net3.1 and net5.0 runtimes, and net6.0 SDK, then invoke the dotnet test and passes all the test assemblies together.

Reply:
Thanks, resolving this.